README.md

```diff
@@ -15,7 +15,7 @@ Get access to this following information:
 
 ## Configure
 
-To confiugure the project, copy `config_example.json` on your `config.json` to add:
+To configure the project, we use npm `config` package, follow `config/default.json` example to define yours with:
```

Comment:
you can add a link to https://www.npmjs.com/package/config ;)

Reply:
Done.